src/zoning-district-class/zoning-district-class.repository.ts

```diff
@@ -17,17 +18,44 @@ export class ZoningDistrictClassRepository {
 
   async findMany(): Promise<FindManyRepo> {
     try {
-      return await this.db.query.zoningDistrictClass.findMany();
+      const result = await this.db.query.zoningDistrictClass.findMany({
+        columns: {
+          id: true,
+          description: true,
+          category: true,
+          color: true,
+          url: true,
+        },
+      });
+
+      return result.map((zoningDistrictClass: any) => ({
+        ...zoningDistrictClass,
+        category: zoningDistrictClass.category as ZoningDistrictClassCategory,
+      }));
```

Comment:
The  `map` operation on line `31` adds an O(n) operation to achieve the type assertion. I think it would be more efficient to convert the `findMany` operation on line `21` to a `select` statement and perform the type coercion directly in there. That way, we achieve the same result in a single operation.

```typescript
return this.db.select([COLUMNS including casting the category]).from(zoningDistrictClass)...
```